ext/PerlIO-encoding/encoding.xs

```diff
@@ -55,7 +55,7 @@ typedef struct {
 
 #define NEEDS_LINES	1
 
-static const MGVTBL PerlIOEncode_tag = { 0, 0, 0, 0, 0, 0, 0, 0 };
+static const MGVTBL PerlIOEncode_tag = {};
```

Comment:
This still needs to be `{0}` [per MSVC](https://github.com/Perl/perl5/actions/runs/8340362405/job/22824615617?pr=22086#step:4:3002).